NetKet/Hilbert/hilbert_index.hpp

```diff
@@ -24,37 +24,20 @@
 #include <nonstd/span.hpp>
 #include <vector>
 
-#include "Hilbert/abstract_hilbert.hpp"
 #include "Utils/next_variation.hpp"
 
 namespace netket {
 
 class HilbertIndex {
-  const std::vector<double> localstates_;
```

Comment:
Do we have code style guidelines for where to declare private variables in classes?

Reply:
According to https://google.github.io/styleguide/cppguide.html#Declaration_Order:
> A class definition should usually start with a public: section, followed by protected:, then private:. Omit sections that would be empty.